lib/ansible/modules/network/netvisor/pn_access_list_ip.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Pluribus Networks
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: pn_access_list_ip
+author: "Pluribus Networks (rpachi@pluribusnetworks.com)"
+version_added: "2.8"
+short_description: CLI command to add/remove access-list-ip
+description:
+  - This modules is used to dd IPs associated with access list
+    and remove IPs associated with access list.
+options:
+  pn_cliswitch:
+    description:
+      - Target switch to run the CLI on.
+    required: False
+    type: str
+  state:
+    description:
+      - State the action to perform. Use 'present' to add access-list-ip and
+      'absent' to remove access-list-ip.
+    required: True
+    choices: [ "present", "absent"]
+  pn_ip:
+    description:
+      - IP associated with the access list
+    required: false
+    type: str
+    default: '::'
+  pn_name:
+    description:
+      - Access List Name
+    required: false
+    type: str
+"""
+
+EXAMPLES = """
+- name: access list ip functionality
+  pn_access_list_ip:
+    pn_cliswitch: "sw01"
+    pn_name: "foo"
+    pn_ip: "172.16.3.1"
+    state: "present"
+
+- name: access list ip functionality
+  pn_access_list_ip:
+    pn_cliswitch: "sw01"
+    pn_name: "foo"
+    pn_ip: "172.16.3.1"
+    state: "absent"
+"""
+
+RETURN = """
+command:
+  description: the CLI command run on the target node.
+  returned: always
+  type: string
+stdout:
+  description: set of responses from the access-list-ip command.
+  returned: always
+  type: list
+stderr:
+  description: set of error responses from the access-list-ip command.
+  returned: on error
+  type: list
+changed:
+  description: indicates whether the CLI caused changes on the target.
+  returned: always
+  type: bool
+"""
+
+import shlex
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.network.netvisor.pn_nvos import pn_cli, run_cli
+
+
+def check_cli(module, cli):
+    """
+    This method checks for idempotency using the access-list-ip-show command.
+    If ip  exists, return IP_EXISTS as True else False.
+    :param module: The Ansible module to fetch input parameters
+    :param cli: The CLI string
+    :return Global Booleans: IP_EXISTS
+    """
+    name = module.params['pn_name']
+    ip = module.params['pn_ip']
+
+    show = cli + \
```

Comment:
```python
show = [cli, 'access-list-ip-show', 'name', name, 'format', 'ip', 'no-show-headers'] 
out = module.run_command(show, use_unsafe_shell=True)[1]
```

Reply:
ok let change and test and send it for review.

Reply:
Above show which you have suggested wont split my cli parameter hence i have used 

Hence i have concatenated the string before using it in run_command.

  out = module.run_command(cli.split(), use_unsafe_shell=True)[1]